docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash --login
+# The --login ensures the bash configuration is loaded,
+# enabling Conda..
+conda activate ldar_sim_env
+pip install orbit_predictor
+pip install timezonefinder
+export AWS_KEY=AKIAVZXPNFOVPRYFUWWR
+export AWS_SEC=ZZl2zdpjm/jrcCNVPf7OjNYy58TIMWG11VXwKJMj
```

Comment:
Are these OK to put on the put on the internet?

Reply:
Since there is no need to keep AWS credentials, I removed them. But, generally, I am controlling AWS access to the S3 buckets and there is no risk here.